### PR TITLE
chore(menu): removed dark theme

### PR DESCRIPTION
**Pull request by moog16**

### CODE REVIEW

File: packages/mdc-menu/mdc-menu.scss [hunk 1]
```diff
@@ -16,7 +16,6 @@
 
 @import "@material/animation/variables";
 @import "@material/elevation/mixins";
-@import "@material/theme/mixins";
```
Comment (kfranqueiro) [inline packages/mdc-menu/mdc-menu.scss]: Don't remove this; `mdc-theme-prop` is still used in this file.

File: packages/mdc-menu/mdc-menu.scss [hunk 2]
```diff
@@ -118,38 +114,20 @@ $mdc-menu-scale-duration: .12s;
...
   .mdc-list-item__graphic {
     color: rgba(black, .54);
-
-    @include mdc-theme-dark(".mdc-menu", true) {
-      color: rgba(white, .54);
-    }
   }
 
   .mdc-list-item[aria-disabled="true"] {
     @include mdc-theme-prop(color, text-disabled-on-light);
```
Comment (acdvorak) [inline packages/mdc-menu/mdc-menu.scss]: `text-disabled-on-background`

File: packages/mdc-menu/mdc-menu.scss [hunk 3]
```diff
@@ -118,38 +114,20 @@ $mdc-menu-scale-duration: .12s;
...
   }
 
-  .mdc-list-divider {
-    @include mdc-theme-dark(".mdc-menu", true) {
-      border-color: rgba(white, .12);
-    }
-  }
-
   .mdc-list-item__graphic {
     color: rgba(black, .54);
```
Comment (acdvorak) [inline packages/mdc-menu/mdc-menu.scss]: Replace this line with:

`@include mdc-theme-prop(color, text-secondary-on-background);`

File: packages/mdc-menu/mdc-menu.scss [hunk 4]
```diff
@@ -27,10 +27,6 @@ $mdc-menu-scale-duration: .12s;
 .mdc-menu {
```
Comment (acdvorak) [inline packages/mdc-menu/mdc-menu.scss]: Can we change `background-color: white` to `background-color: $mdc-theme-background`?

(We'll need to add `@import "@material/theme/variables"` to the top of this file)

File: packages/mdc-menu/mdc-menu.scss (hunk 3 above)
Comment (moog16) [reply packages/mdc-menu/mdc-menu.scss]: I traced it back through mdc-theme, so it should be the same, but we don't have a demo to showcase this. Should we add a menu item with a graphic?

**Pull request merged**